app/models/restaurant.rb

```diff
@@ -0,0 +1,37 @@
+# == Schema Information
+#
+# Table name: restaurants
+#
+#  id                           :bigint           not null, primary key
+#  name                         :string(40)
+#  cuisine                      :string(20)
+#  rating                       :float
+#  is_10_bis                    :boolean
+#  address                      :string(60)
+#  max_delivery_time_in_minutes :integer
+#  created_at                   :datetime         not null
+#  updated_at                   :datetime         not null
+#
+
+class Restaurant < ApplicationRecord
+  # Associations
+  has_many :reviews
+
+  # Scopes
+  scope :sorted, lambda { order("name ASC") }
+
+  # Validations
+  validates :name,
+            :presence => true,
+            :length => {:maximum => 40}
+  validates :cuisine,
+            :presence => true,
```

Comment:
And just FYI, if you come to need it, you can also validate multiple model attributes together `  validates :name, : rating, :address, presence: true`

Reply:
Yes I saw it, but I think it's more readable to write the validations of each attribute separately